src/Application/Domain/Entities/Genre.cs

```diff
@@ -12,7 +12,7 @@ public Genre(string name)
         Name = name;
     }
     public Guid Id { get; private set; }
-    public string Name { get; private set; }
+    public string Name { get; set; }
```

Comment:
manten privado el set y agrega un metodo para actualizar este valor
```suggestion
    public void UpdateName(string name){
        // posibles validaciones
        Name = name
    }
```